app/templates/admin/sales/invoices.hbs

```diff
@@ -1,7 +1,7 @@
 <div class="ui unstackable grid">
   <div class="row">
     <div class="column">
-      <h2 class="ui header">
+      <h2 class="ui header gray-text">
```

Comment:
This still means ui header everywhere is light gray. Why not just add gray color to ui.header class without !important?